ivy/functional/backends/paddle/experimental/losses.py

```diff
@@ -0,0 +1,41 @@
+# global
+from typing import Optional
+import paddle
+import paddle.nn.functional as F
```

Comment:
same instructions for paddle as given below for torch

Reply:
You haven't addressed this yet, you have to use paddle the same way you used torch